src/Mvc/samples/MvcSandbox/Startup.cs

```diff
@@ -66,13 +70,10 @@ public void Configure(IApplicationBuilder app)
 
                 builder.MapControllers();
                 builder.MapRazorPages();
-
-                builder.MapFallbackToController("Index", "Home");
+                builder.MapComponentHub<MvcSandbox.Components.App>("app");
```

Comment:
I think you need to pass the path here too.

Reply:
Path to what? This is working

Reply:
Actually, nevermind. This should work just fine.